tests/test_src.py

```diff
@@ -1,25 +1,21 @@
 from app import set_prices
-from app import app
+
 
 def test_set_prices_sorting_works():
-    #
+    # Sortowanie
```

Comment:
please, keep the comments in English, I cannot understand it now:)

Reply:
learn polish nub. Polska górom